pkg/resolution/resolver/hub/resolver.go

```diff
@@ -359,8 +359,8 @@ func validateParams(ctx context.Context, paramsMap map[string]string, tektonHubU
 		missingParams = append(missingParams, ParamVersion)
 	}
 	if kind, ok := paramsMap[ParamKind]; ok {
-		if kind != "task" && kind != "pipeline" {
-			return errors.New("kind param must be task or pipeline")
+		if kind != "task" && kind != "pipeline" && kind != "stepaction" {
+			return errors.New("kind param must be either a task, pipeline or a stepaction")
```

Comment:
Optional suggestion: I like they way this was addressed [in the Cluster Resolver](https://github.com/tektoncd/pipeline/blob/23b3deefcea58e469e5775c4b39f6657dc80b538/pkg/resolution/resolver/cluster/resolver.go#L371-L373), with a const list of `supportedKinds`. Any objection to following that pattern?